homeassistant/const.py

```diff
@@ -564,7 +564,13 @@ class UnitOfElectricPotential(StrEnum):
 CURRENCY_EURO: Final = "€"
 CURRENCY_DOLLAR: Final = "$"
 CURRENCY_CENT: Final = "¢"
-
+CURRENCY_POUNDS: Final = "£"
+CURRENCY_YEN: Final = "¥"
+CURRENCY_LIRA: Final = "₺"
+CURRENCY_FRANC: Final = "₣"
+CURRENCY_ROUBLE: Final = "₽"
+CURRENCY_ETHER: Final = "Ξ"
+CURRENCY_BTC: Final = "₿"
```

Comment:
We don't add common constants unless those strings are already in use in several places in the application.

Reply:
Here's the script that generates the allowed currencies, in the backend:

https://github.com/home-assistant/core/blob/dev/script/currencies.py

There may also be code in the frontend for the available currencies to select in the GUI.